xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -648,16 +648,43 @@ AVDictionary *CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
       std::string name = it->first; StringUtils::ToLower(name);
       const std::string &value = it->second;
 
-      if (name == "seekable")
-        av_dict_set(&options, "seekable", value.c_str(), 0);
-      else if (name == "user-agent")
+      if (name == "user-agent")
       {
         av_dict_set(&options, "user-agent", value.c_str(), 0);
         hasUserAgent = true;
       }
+      
+      // these are the officially supported headers from ffmpeg
+      else if (name == "seekable" || name == "chunked_post" || name == "content_type" || name == "metadata" || name == "offset" ||
+        name == "multiple_requests" || name == "post_data" || name == "mime_type" || 
+        name == "icy" || name == "icy_metadata_headers" || name == "icy_metadata_packet" || name == "method" ||
+        name == "auth_type" || name == "none" || name == "basic" || name == "send_expect_100" || name == "location" || 
+        name == "end_offset" || name == "http_proxy" || name == "reconnect" || name == "reconnect_at_eof" || name == "reconnect_streamed" ||
+        name == "reconnect_delay_max" || name == "listen" || name == "resource" || name == "reply_code")
+      {
+        CLog::Log(LOGDEBUG, "CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput() adding ffmpeg option '%s: %s'", it->first.c_str(), value.c_str());
+        av_dict_set(&options, it->first.c_str(), value.c_str(), 0);
+      }
+      // other standard headers (see https://en.wikipedia.org/wiki/List_of_HTTP_header_fields)
+      else if (name == "accept" || name == "accept-language" || name == "accept-datetime" || 
+        name == "authorization" || name == "cache-control" || name == "connection" || name == "cookie" || name == "content-md5" || 
+        name == "content-type" || name == "date" || name == "expect" || name == "forwarded" || name == "from" || name == "host" || name == "if-match" || 
```

Comment:
btw content-type is directly connected with post_data couse its used with POST request